docs/setting-up/client/mongodb.md

```diff
@@ -245,6 +245,46 @@ pmm-admin remove mongodb SERVICE_NAME
 
 - `SERVICE_NAME`: The name the service was added as. (Find it with `pmm-admin list`.)
 
+## Advanced Options
+
+### Enabling all collectors
+
+By default, PMM starts the MongoDB exporter using these parameters:
+
+|Parameter|Description|
+|-----|-----|
+|`--collector.diagnosticdata`|Enable `getDiagnosticData` collector|
+|`--collector.replicasetstatus`|Enable `replSetGetStatus` collector|
+|`--discovering-mode`|Auto discover all databases and collections|
+|`--compatible-mode`|Enable MongoDB exporter v1 compatible metric names|
+|`--mongodb.global-conn-pool`|Use a single connection to the DB instead of connecting in each scrape|
+|`--collector.collstats-limit=200`|Enable collStats and indexStats only if there are less than n collections in total|
+
+To enable all collectors, add `--enable-all-collectors` to the `pmm-admin add mongodb` command.
+This will enable: `collstats`, `dbstats`, `indexstats` and the `topmetrics` collectors.
+
+### Disable some collectors.
+
+To enable some collectors but not all of them, it is posible to combine `--enable-all-collectors` along with `--disable-collectors`.  
+For example, if you want all collectors but `topmetrics`, specify:
+```
+--enable-all-collectors --disable-collectors=topmetrics
+```
+
+### Limiting dbStats, collStats and indexStats
+
+By default, when all collectors are enabled, the collStats and indexStats collectors are enabled only if there are less than
+200 collections across all databases, excluding the collections in the system databases `admin`, `config`, `local`.  
```

Comment:
By default, all the collectors are enabled. However, collStats and indexStats are enabled if there are less than 200 collections across all the databases. This excludes the collections in the system databases admin, config, and local.